### ISSUE TITLE
Warn on >>>()

### ISSUE THREAD

**Issue by unhammer** [koalaman/shellcheck#1743]
#### For new checks and feature suggestions
- [x] shellcheck.net (i.e. the latest commit) currently gives no useful warnings about this
- [x] I searched through https://github.com/koalaman/shellcheck/issues and found 467 open and 1255 closed ie. I don't know how to search for this

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

{ echo hi1; echo hi2 >&2; echo error1; echo error2 >&2; } 2>>>(grep -v error)

```

#### Here's what shellcheck currently says:

zilch

#### Here's what I wanted or expected to see:

`2>>>()` will fail on older bash versions, use `2> >()` instead

(Also, perhaps it should also warn on `2>> >()` has a useless `>` – or does that ever make sense?)

**Comment by koalaman** [comment]
Which versions of Bash will this fail on? I tried several versions going back to version 1.14.7, and couldn't find any with different behavior.

**Comment by unhammer** [comment]
Oh, sorry, it's `&>>>` that fails, not `2>>>`. But I'd say both are weird enough to warrant a hint …

```
$ bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin18)
Copyright (C) 2007 Free Software Foundation, Inc.
$ { echo hi1; echo hi2 >&2; echo error1; echo error2 >&2; } &>>>(grep -v err) 
-bash: syntax error near unexpected token `>>'
```
